examples/stress_tests/particles.rs

```diff
@@ -139,7 +139,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let mut session_builder = SessionBuilder::<Config>::new()
         .with_num_players(num_players)
         .with_desync_detection_mode(desync_mode)
-        .with_max_prediction_window(args.max_prediction)?
```

Comment:
I guess this became non-optional in ggrs? Seems unrelated to the bevy upgrade, not necessarily a problem, just wanted to check.

Reply:
Yeah, but the way the ggrs dependency is set up I think I upgraded that by acciden.